### PR TITLE
Add minFeeMultiplier

### LINKED ISSUE #326: How useful is the /network/fees/transaction endpoint when it returns all zeros
When there is no transaction activity on the blockchain, the network/fees/transaction endpoint returns all zeros.  These values cannot be used on the testnet where the default minFeeMultiplier = 100 and transactions will get dropped.

{"averageFeeMultiplier":0,"medianFeeMultiplier":0,"highestFeeMultiplier":0,"lowestFeeMultiplier":0}

		server.get('/network/fees/transaction', (req, res, next) => {
			const numBlocksTransactionFeeStats = services.config.numBlocksTransactionFeeStats || 1;
			return db.latestBlocksFeeMultiplier(numBlocksTransactionFeeStats).then(feeMultipliers => {
				res.send({
					averageFeeMultiplier: Math.floor(average(feeMultipliers)),
					medianFeeMultiplier: Math.floor(median(feeMultipliers)),
					highestFeeMultiplier: Math.max(...feeMultipliers),
					lowestFeeMultiplier: Math.min(...feeMultipliers)
				});
				next();
			});
		});

### PR DESCRIPTION
**Pull request by rg911**
- Added nodePropertyFilePath in apiNode in rest.json
- Renamed & moved network properties filePath to apiNode in rest.json
- Added minFeeMultiplier to fees endpoint
fixed #326

### CODE REVIEW

File: rest/src/routes/networkRoutes.js [hunk 1]
```diff
@@ -62,12 +68,15 @@ module.exports = {
...
 				res.send({
-					averageFeeMultiplier: Math.floor(average(feeMultipliers)),
-					medianFeeMultiplier: Math.floor(median(feeMultipliers)),
-					highestFeeMultiplier: Math.max(...feeMultipliers),
-					lowestFeeMultiplier: Math.min(...feeMultipliers)
+					averageFeeMultiplier: Math.floor(average(feeMultipliers[1])),
+					medianFeeMultiplier: Math.floor(median(feeMultipliers[1])),
+					highestFeeMultiplier: Math.max(...feeMultipliers[1]),
+					lowestFeeMultiplier: Math.min(...feeMultipliers[1]),
+					minFeeMultiplier: Number(feeMultipliers[0].node.minFeeMultiplier.replace(`'`, ''))
```
Comment (fboucquez) [inline rest/src/routes/networkRoutes.js:79]: nodeMinFeeMultiplier? This value would be different for different nodes.

File: rest/resources/rest.json [hunk 2]
```diff
@@ -40,7 +40,9 @@
     "timeout": 1000,
     "tlsClientCertificatePath": "/",
     "tlsClientKeyPath": "/",
-    "tlsCaCertificatePath": "/"
+    "tlsCaCertificatePath": "/",
+    "networkPropertyFilePath": "/",
+    "nodePropertyFilePath": "/"
```
Comment (fboucquez) [inline rest/resources/rest.json:45]: Remove old propertiesFilePath from network?

File: rest/src/routes/networkRoutes.js (hunk 1 above)
Comment (rg911) [reply rest/src/routes/networkRoutes.js:79]: we can leave it like this, and comment in the openAPI spec I guess?

File: rest/resources/rest.json (hunk 2 above)
Comment (rg911) [reply rest/resources/rest.json:45]: the old one is not even in the config, :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [v2.2.2] - 13-Jan-2020
+
+### Added
+
+- `minFeeMultiplier` from the current connected Node in transaction fees endpoint
+- Added new `nodeProertyFilePath` in `rest.json`.
```
Comment (fboucquez) [inline CHANGELOG.md:12]: nodeProertyFilePath typo

File: rest/src/index.js [hunk 4]
```diff
@@ -69,7 +69,7 @@ const loadConfig = () => {
 	configFiles.forEach(configFile => {
 		winston.info(`loading config from ${configFile}`);
 		const partialConfig = JSON.parse(fs.readFileSync(configFile, 'utf8'));
-
+		console.log('AAAAAAAAAAAAAAAAAAAAAAAa',partialConfig)
```
Comment (fboucquez) [inline rest/src/index.js:72]: remove

File: rest/src/routes/networkRoutes.js (hunk 1 above)
Comment (fboucquez) [reply rest/src/routes/networkRoutes.js:79]: Ok, fair, keep minFeeMultiplier

File: rest/src/routes/networkRoutes.js [hunk 5]
```diff
@@ -62,12 +68,15 @@ module.exports = {
 
 		server.get('/network/fees/transaction', (req, res, next) => {
 			const numBlocksTransactionFeeStats = services.config.numBlocksTransactionFeeStats || 1;
-			return db.latestBlocksFeeMultiplier(numBlocksTransactionFeeStats).then(feeMultipliers => {
+			const nodeProperties = readAndParseNodePropertiesFile();
```
Comment (fboucquez) [inline rest/src/routes/networkRoutes.js:71]: Property files should be cached somewhere and not lead each rest request.

Create a const on top of the route definition?

Comment (fboucquez) [reply rest/src/routes/networkRoutes.js:71]: Move `const nodeProperties = readAndParseNodePropertiesFile(); `aboce server.get(

**Review by fboucquez — CHANGES_REQUESTED** [review]

**Pull request merged**